clio-server/src/main/scala/org/broadinstitute/clio/server/service/GvcfService.scala

```diff
@@ -21,57 +12,10 @@ import scala.concurrent.{ExecutionContext, Future}
 class GvcfService(
   persistenceService: PersistenceService,
   searchService: SearchService
-)(
-  implicit executionContext: ExecutionContext,
-  index: ElasticsearchIndex[GvcfIndex.type]
-) {
-
-  def upsertMetadata(
-    transferKey: TransferGvcfV1Key,
-    transferMetadata: TransferGvcfV1Metadata
-  ): Future[UpsertId] = {
-    val updatedTransferMetadata = transferMetadata.copy(
-      documentStatus = transferMetadata.documentStatus.orElse(Some(DocumentStatus.Normal))
-    )
-
-    persistenceService
-      .upsertMetadata(
-        transferKey,
-        updatedTransferMetadata,
-        GvcfService.v1DocumentConverter,
-        ElasticsearchIndex.Gvcf
-      )
-  }
-
-  def queryMetadata(
-    transferInput: TransferGvcfV1QueryInput
-  ): Source[Json, NotUsed] = {
-    val transferInputNew =
-      transferInput.copy(documentStatus = Option(DocumentStatus.Normal))
-    queryAllMetadata(transferInputNew)
-  }
-
-  def queryAllMetadata(
-    transferInput: TransferGvcfV1QueryInput
-  ): Source[Json, NotUsed] = {
-    searchService.queryMetadata(
-      transferInput,
-      GvcfService.v1QueryConverter
+)(implicit executionContext: ExecutionContext)
+    extends IndexService[GvcfIndex.type](
```

Comment:
Do you need the explicit type param any more? I think it was required before because you also had `D`, which wasn't inferrable from any of the type parameters.